src/listeners/ready.ts

```diff
@@ -1,16 +1,9 @@
 import type { ListenerOptions, Piece, PieceContext } from '@sapphire/framework';
 import { Listener, Store } from '@sapphire/framework';
```

Comment:
Why are we still using Sapphire's listener?

Reply:
For the record we removed this file after deciding we don't use the store feature